packages/block-library/src/search/edit.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { RichText, InspectorControls } from '@wordpress/editor';
+import { PanelBody, TextControl } from '@wordpress/components';
+
+export default function SearchEdit( { attributes, setAttributes } ) {
+	const { label, placeholder } = attributes;
+
+	return (
+		<Fragment>
+			<div className="wp-block-search">
+				<RichText
```

Comment:
I think should have an aria-label—other blocks seem to achieve that by using the `placeholder` prop. Also perhaps `multiline={ false }`?

Reply:
Good catch—added the `placeholder` prop so that there's a label.

https://github.com/WordPress/gutenberg/issues/13570 is a ticket that tracks adding `multiline={ false }` which looks like it was removed a while ago. For now, I think it's fine to support line breaks—I can't think of any reason we ought to forbid them! 🙂

Reply:
Was sure I could remember `multiline={ false }` working in the editor, but I must have dreamed it 😴. My feeling is that it's  bit unusual for a label to be multiple lines.

Thanks for the fixes, will re-review in a bit.

Reply:
You didn't dream it! 😀 I think that we removed support for `multiline={ false }` when refactoring how `RichText` works in the lead up to 5.0.

I did a little audit of line breaks in the Search block's attributes and I'm OK with how it is currently:

- `label`: `<br>`s are allowed in a `<label>` so we permit it. This is consistent with e.g. captions in the Image block.
- `placeholder`: `<br>`s or `\n`s are not permitted in a `placeholder=""` so we forbid it.
- `buttonText`: `<br>`s are allowed in a `<button>` so we permit it. This is consistent with e.g. the button text in a Button or File block.